djangoappengine/storage.py

```diff
@@ -114,26 +180,51 @@ def get_valid_name(self, name):
     def get_available_name(self, name):
         return name.replace('\\', '/')
```

Comment:
Nevermind my last comment. The only reason that blobstore worked had to do with the particularities of the way I created blobs.

However, I do have another question (somewhat related). Why is the `get_available_name` method overridden without calling the parent method? The parent method does much more than this, i.e., it makes sure the name is unique.